zcash_primitives/Cargo.toml

```diff
@@ -90,13 +90,15 @@ criterion = "0.4"
 proptest = "1.0.0"
 assert_matches = "1.3.0"
 rand_xorshift = "0.3"
-orchard = { version = "0.3", features = ["test-dependencies"] }
+orchard = { version = "0.3", default-features = false, features = ["test-dependencies"] }
 
 [target.'cfg(unix)'.dev-dependencies]
 pprof = { version = "0.11", features = ["criterion", "flamegraph"] } # MSRV 1.56
 inferno = ">=0.11, <0.11.5" # MSRV 1.59
 
 [features]
+default = ["multicore"]
+multicore = ["orchard/multicore"]
```

Comment:
Document this added feature flag in the changelog (as exposing a way for users to selectively disable multicore support for Orchard proof creation).